docker/Dockerfile

```diff
@@ -1,31 +1,54 @@
-FROM rust:1.39.0 AS builder
-LABEL maintainer="calvin@crypto.com"
+From ubuntu:18.04
+
+RUN apt-get update && apt-get install -y \
+     make \
+     libzmq3-dev \
+     libssl1.1 \
+     curl \
+     libprotobuf-dev \
+     libssl-dev && \
+     rm -rf /var/lib/apt/lists/*
+
+arg BUILD_MODE
+
+# copy the chain binary file
+COPY ./target/${BUILD_MODE}/client-cli /usr/local/bin/
+COPY ./target/${BUILD_MODE}/chain-abci /usr/local/bin/
+COPY ./target/${BUILD_MODE}/client-rpc /usr/local/bin/
+COPY ./target/${BUILD_MODE}/dev-utils /usr/local/bin/
```

Comment:
I think we should use multi-stage build mode, copy binaries from the host machine are error-prone.